lib/optimizely/config_manager/http_project_config_manager.rb

```diff
@@ -105,13 +105,15 @@ def stop!
       @closed = true
     end
 
-    def get_config
+    def config
       # Get Project Config.
 
       # If the background datafile polling thread is running. and config has been initalized,
       # we simply return config.
       # If it is not, we wait and block maximum for @blocking_timeout.
       # If thread is not running, we fetch the datafile and update config.
+      return @config if @closed
```

Comment:
@mjc1283 I think, better attribute name is stopped, since function name is stop. Do you suggest to rename it?
@rashidsp can you please add a comment here, if closed is true, then config is set to nil.

Reply:
@msohailhussain I agree that `stopped` is a better attribute name.